package.json

```diff
@@ -12,10 +12,10 @@
   ],
   "author": "Remind 101 (https://github.com/remind101)",
   "dependencies": {
-    "generic-pool": "^2.0.4",
-    "q": "^1.0.1",
-    "redis": "^0.10.3",
-    "underscore": "^1.6.0"
+    "generic-pool": "~2.4",
+    "q": "~1.0",
+    "redis": "2.6.2",
```

Comment:
also
```
Redis < v. 2.6 is not officially supported anymore and might not work in all cases. Please update to a newer redis version as it is not possible to test for these old versions
```
as of v2.5.0-1. I'm sure we have a newer one, but it wouldn't hurt to double-check